package.json

```diff
@@ -51,8 +51,11 @@
   "dependencies": {
     "axios": "^0.19.0",
     "bootstrap": "^4.3.1",
+    "chia": "0.0.1",
```

Comment:
`chia` like chia pets? Do you mean `chai`?

Also, testing stuff should be installed as a `devDependency`... `npm i -D chai`

Reply:
ch...ch...ch... chia! Good catch Eliot.